src/css/_expand.scss

```diff
@@ -0,0 +1,38 @@
+$box-shadow: 0px 0px 12px rgba(88, 98, 109, 0.07), 0px 2px 4px rgba(88, 98, 109, 0.3);
+$box-shadow-on-hover: 0px 6px 16px rgba(27, 25, 27, 0.16);
+
+details summary {
+  position: relative;
+  display: flex;
+  align-items: center;
+  gap: 24px;
+  padding: 24px;
+  font-family: "OpenSans-Semibold", sans-serif;
+  font-size: 18px;
+  line-height: 24px;
+  color: map-get($monochrome, default);
+  background-color: map-get($monochrome, white);
+  box-shadow: $box-shadow;
+  border-radius: 8px;
+  cursor: pointer;
+  transition: box-shadow 200ms ease, transform 200ms ease;
+
+  &:hover{
+    outline: none;
+    text-decoration: none;
+    color: map-get($monochrome, default);
+    box-shadow: $box-shadow-on-hover;
+    transform: scale(1.01);
+  }
+}
+
+details summary::after{
+  content: url(/static/img/arrow.svg);
+  position: absolute;
+  right: 28px;
+  z-index: 100;
+}
+
+details[open] summary::after {
```

Comment:
Consider nesting that as well.
Just implement breakdown a `details summary {` rule above into two levels.